### ISSUE TITLE
model selection based on test accuracy!

### ISSUE THREAD

**Issue by imtiazziko** [dapengchen123/crf_affinity#6]
Hi, @dapengchen123 , We see that the reported accuracy is based on the model validated on the test set(query_loader,gallery_loader), which is very confusing. If i understand correctly, it seems like you violate the training protocol for person reid task where the norm is to utilize the validation set(disjoint from the test set) for the model selection. Can you please update the issue regarding this. 
This will be much appreciated. Thank you so much.

**Comment by dapengchen123** [comment]
Actually, the testing part in the code is not used for model selection, it just monitors how performance changes with epoch increase.  We just take the model of the last epoch for evaluation, you can see the performance is stable over the last several epochs.

**Comment by imtiazziko** [comment]
Thanx
